src/Pages/Playground/components/playgrounApiDocs.tsx

```diff
@@ -0,0 +1,587 @@
+                              "type": "number"
+                          },
+                          "app_markup_value": {
+                              "description": "The markup the client paid in the app developer's currency",
+                              "type": "number"
+                          },
+                          "client_currcode": {
+                              "description": "Currency code of the client",
+                              "type": "string"
+                          },
+                          "client_loginid": {
+                              "description": "Login ID of the client",
+                              "type": "string"
+                          },
+                          "dev_currcode": {
+                              "description": "Currency code of the app developer",
+                              "type": "string"
+                          },
+                          "dev_loginid": {
+                              "description": "Login ID of the app developer",
+                              "type": "string"
+                          },
+                          "transaction_id": {
+                              "description": "The transaction ID. Every contract (buy or sell) and every payment has a unique ID.",
+                              "type": "integer",
+                              "examples": [
+                                  10867502908
+                              ]
+                          },
+                          "transaction_time": {
+                              "description": "The epoch value of purchase time of transaction",
+                              "type": "string"
+                          }
+                      }
+                  }
+              }
+          }
+      },
+      "echo_req": {
+          "description": "Echo of the request made.",
+          "type": "object"
+      },
+      "msg_type": {
+          "description": "Action name of the request made.",
+          "type": "string",
+          "enum": [
+              "app_markup_details"
+          ]
+      },
+      "req_id": {
+          "description": "Optional field sent in request to map to response, present only when request contains `req_id`.",
+          "type": "integer"
+      }
+  }
+}
+
+const Box: Component<BoxType> = ({ content, activeClass, nested }) => {
+  const [showSignatureBox, setShowSignatureBox] = createSignal(false);
+  const [showSourceButton, setShowSourceButton] = createSignal(false);
+
+  const strDescription = content?.description?.replace(/(`.`)/g, (match:string) => `<code class="bg-[rgba(255,255,255,.16)] text-[#fff] text-[14px] mr-1.5 px-2 py-1.5 font-normal rounded">${match}</code>`)?.replace(/`/g ,'');
+
+  let expand_ref: any, source_ref: any;
+
+  const mouseHoverFunc = (show: number) => {
+    expand_ref.style.opacity = show;
+    source_ref.style.opacity = show;
```

Comment:
or maybe there is mutable ref in solid just as useRef? Or is it recommended to make mutable ref object by just setting empty variables?

Reply:
Yeah, right. it is used as Ref like useRef in react. As I found under solidJs the ref should be defined in this way.
source: https://www.solidjs.com/docs/latest/api#ref

Reply:
thanks for the pointer. Looks clean and reusing JS syntax, looks nice